packages/nest/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './open-feature.module';
```

Comment:
You might want to consider adding something like this: https://github.com/open-feature/js-sdk/blob/c7098c365a3723c12eb3b1b35b1f5c58e0cc231f/packages/react/src/index.ts#L3C1-L3C1

This allows users to import all the things they need from the Nest SDK (it just re-exports the exports of the peer). There is no runtime impact, all the same modules are resolved, etc, it's just a nice experience to be able to import everything from the "highest level" sdk.

Reply:
Ya, I thought we might not need, other Nest Packages also do not do this but I will add it.